Owncloud iOs Client/DataBase/Queries/ManageSharesDB.m

```diff
@@ -164,6 +163,8 @@ + (NSMutableArray*) getSharesByFolder:(FileDto *) folder {
             sharedDto.shareWithDisplayName = [rs stringForColumn:@"share_with_display_name"];
             sharedDto.isDirectory = [rs boolForColumn:@"is_directory"];
             sharedDto.idRemoteShared = [rs intForColumn:@"id_remote_shared"];
```

Comment:
I rather leave it for next release too, qa already approved that, let's made the release..